README.md

```diff
@@ -126,7 +126,7 @@ thor docs:clean     # Delete documentation packages
 thor console        # Start a REPL
 thor console:docs   # Start a REPL in the "Docs" module
 
-# Tests can be run quickly from within the console using the "test" command. 
+# Tests can be run quickly from within the console using the "test" command.
```

Comment:
Ups! My editor auto-removed some blank spaces, is that okay? I can revert it if needed.

Reply:
That's fine :)